deepmd/training/data/data_3/type.raw

```diff
@@ -0,0 +1,192 @@
+0
```

Comment:
Let's just put everything inside `data` to LFS, pls. You'd need to re-create the commits instead of adding to this tree.